sbiexpt/analytic_bijector.py

```diff
@@ -0,0 +1,55 @@
+
+
+class SmoothCompactBumpFunction(tfp.bijectors.Bijector):
+
+    def __init__(self, a, b, c,  name = 'SmoothCompactBumpFunction'):
+      super(SmoothCompactBumpFunction, self).__init__(forward_min_event_ndims=0, name = name)
+      self.a = a
+      self.b = b
+      self.c = c
+
+    def _forward(self, x):
+      return f(x, self.a, self.b, self.c)
+
+    def _inverse(self, y):
+      return inv_f(y,self.a, self.b, self.c)
+
+    def _forward_log_det_jacobian(self, x):
+      grad = gradf(x,self.a,self.b,self.c)
+      return jnp.atleast_2d(jnp.log(jnp.abs(jnp.prod(grad, axis=1)))).T
```

Comment:
hum that looks suspicious, the fact that you need a .T at the end.
Note that this may not be the most stable way of computing a log det in the general case (although here this should be fine). For stability we usually prefer to use this function: https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.slogdet.html